src/driver-asus.c

```diff
@@ -37,16 +37,17 @@
 
 #include "asus.h"
 
-/* ButtonMapping configuration property defaults, stores indices for ASUS_BUTTON_MAPPING */
-static int8_t ASUS_BUTTON_MAPPING_INDEX[] = {
-	0,  /* left */
-	1,  /* right (button 3 in xev) */
-	2,  /* middle (button 2 in xev) */
-	6,  /* wheel up */
-	7,  /* wheel down */
-	5,  /* DPI */
-	3,  /* backward */
-	4,  /* forward */
+/* ButtonMapping configuration property defaults */
+static int8_t ASUS_CONFIG_BUTTON_MAPPING[] = {
+	0xf0,  /* left */
+	0xf1,  /* right (button 3 in xev) */
+	0xf2,  /* middle (button 2 in xev) */
+	0xe4,  /* backward */
+	0xe5,  /* forward */
+	0xe6,  /* DPI */
+	0xe8,  /* wheel up */
+	0xe9,  /* wheel down */
```

Comment:
This problem was here even before this PR, but these values do not fit in 8 bytes, see compiler warning:

```
../src/driver-asus.c:42:9: warning: conversion from ‘int’ to ‘signed char’ changes value from ‘240’ to ‘-16’ [-Wconversion]
   42 |         0xf0,  /* left */
      |
```